### PR TITLE
AMP-77128 Add explanation for estimated debug delivery metrics

### PR DESCRIPTION
**Pull request by daniel-beer**
# Amplitude Developer Docs PR

## Description

Add explanation for how debug delivery metrics are estimated for GA4 destinations.

## Deadline

ASAP

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Review by justin-fiedler — APPROVED** [review]

**Comment by justin-fiedler** [comment]
Please fix [MegaLinter errors](https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/actions/runs/5247951477/jobs/9478795768) 

image

**Comment by daniel-beer** [comment]
> 2

Will do. I can't seem to run mkdocs locally so was waiting for the PR checks

File: includes/debug-delivery-metrics.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Delivery metrics
+
+The events delivered, not delivered, and delivery rate metrics for this destination are an estimate based on a sample of events sent to the Google Analytics 4 Measurement Protocol validation endpoint. In addition to sending all events to the ingestion endpoint, a sample of up to the first 100 events (and 100 user property changes, if "Send Users" is enabled) are sent to the validation endpoint every 10 minutes. The metrics shown in Amplitude are scaled from the sample of events, sent to the validation endpoint, to the total volume of events sent to the ingestion endpoint. If the delivery rate is below 100%, the counts of events delivered shown in Amplitude may not match exactly with the number of events seen in Google Analytics 4. Latency metrics are still computed exclusively from events sent to the ingestion endpoint.
```
Comment (sina-jamshidi) [inline includes/debug-delivery-metrics.md:3]: Should this maybe have a link?
>Google Analytics 4 Measurement Protocol validation endpoint

**Review by sina-jamshidi — APPROVED** [review]

File: includes/debug-delivery-metrics.md (hunk 1 above)
Comment (daniel-beer) [reply includes/debug-delivery-metrics.md:3]: Sure. I can add links for both the validation and ingestion endpoints.

**Pull request merged**